native_client/kenlm/util/file.cc

```diff
@@ -540,7 +540,7 @@ std::string DefaultTempDirectory() {
   const char *const vars[] = {"TMPDIR", "TMP", "TEMPDIR", "TEMP", 0};
   for (int i=0; vars[i]; ++i) {
     char *val =
-#if defined(_GNU_SOURCE)
+#if defined(_GNU_SOURCE) && defined(__GLIBC_PREREQ)
```

Comment:
This change needs to be documented in README.mozilla, probably easiest way would be as a patch file.

Reply:
Done

Reply:
Done